apps/frontend/src/libs/components/sidebar/sidebar-item/sidebar-item.tsx

```diff
@@ -0,0 +1,23 @@
+import { Icon, Link } from "~/libs/components/components.js";
+import { type IconName } from "~/libs/components/icon/types/types.js";
+import { type AppRoute } from "~/libs/enums/enums.js";
+import { type ValueOf } from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	href: "#" | ValueOf<typeof AppRoute>;
+	icon: IconName;
+	label: string;
+};
+
+const SidebarItem = ({ href, icon, label }: Properties): JSX.Element => (
+	<li className={styles["sidebar-item"]}>
+		<Link to={href}>
+			<Icon height={17} name={icon} width={17} />
```

Comment:
In the design, I see that the icons take 20px. You need to download them with the frame they are included with, because this frame adjust their dimensions to be the same and equal across the project. Also we for now we don't need any icons except for project, so please remove the rest ones

![Screenshot 2024-08-23 at 11 43 50](https://github.com/user-attachments/assets/4cacf920-acbf-4c01-9477-ff08307a0d03)